services/order.py

```diff
@@ -0,0 +1,29 @@
+from django.db import transaction
+from django.db.models import QuerySet
+from django.contrib.auth import get_user_model
+from db.models import Order, Ticket
+
+def create_order(tickets: list[dict], username: str, date: str = None) -> Order:
+    user = get_user_model().objects.get(username=username)
+
+    with transaction.atomic():
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+            order.save(update_fields=["created_at"])
+
+        for ticket in tickets:
+            Ticket.objects.create(
+                order=order,
+                movie_session_id=ticket["movie_session"],
+                row=ticket["row"],
+                seat=ticket["seat"]
+            )
+
+        return order

```

Comment:
Code Style: It's better to use one style of quotes in your code. You are using both single and double quotes. I recommend using double quotes as they are more visually distinct and easier to spot. For example, replace `update_fields=['created_at']` with `update_fields=["created_at"]`.